api/backend.go

```diff
@@ -299,26 +273,18 @@ func (b *StatusBackend) registerHandlers() error {
 		return b.AccountManager().Accounts()
 	})
 
-	rpcClient.RegisterHandler(params.SendTransactionMethodName, func(ctx context.Context, rpcParams ...interface{}) (interface{}, error) {
-		txArgs, err := transactions.RPCCalltoSendTxArgs(rpcParams...)
-		if err != nil {
-			return nil, err
-		}
-
-		hash, err := b.SendTransaction(ctx, txArgs)
-		if err != nil {
-			return nil, err
-		}
+	rpcClient.RegisterHandler(params.SendTransactionMethodName, unsupportedMethodHandler)
 
-		return hash.Hex(), err
-	})
-
-	rpcClient.RegisterHandler(params.PersonalSignMethodName, b.personalAPI.Sign)
-	rpcClient.RegisterHandler(params.PersonalRecoverMethodName, b.personalAPI.Recover)
+	rpcClient.RegisterHandler(params.PersonalSignMethodName, unsupportedMethodHandler)
```

Comment:
Because we want to guard calling it with a password. It's not supported to be called directly, only via a binding. Maybe we can add a comment explaining that.